whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/VersionInformation.java

```diff
@@ -41,4 +45,12 @@ public UUID getSessionID() {
         return sessionID;
     }
 
+    public NrtmDocumentType getType() {
+        return type;
+    }
+
+    public VersionInformation increment() {
+        return new VersionInformation(0L, this.source, this.version + 1, this.sessionID, this.type);
+    }
```

Comment:
I would prefer to not have this logic here and keep it as plain pojo. This will be used for delta files as well and version increment will depend of documentType.

Reply:
i'm not really happy with this either. let's figure out a better place for it. but actually it will /only/ be used for delta files -- snapshots don't increment the version, they just use the one from the last delta. if there are no deltas, then we don't publish a new snapshot (though i think we might still produce a new update notification for the same one).

Reply:
When we have a logic in place for delta file I think we will have a clearer picture. Lets resolve this in later issues

Reply:
done